linux/tarball.tw2

```diff
@@ -0,0 +1,30 @@
+::Linux tarball
+# Getting the Executable
+
+## Download the archive
+
+If you know your process architecture (32-bit vs 64-bit vs
+ARM), download the appropriate archive from the [releases
+page](https://github.com/exercism/cli/releases/latest).
+
+## Extract from the archive
+
+Once you have the archive downloaded you
+need to extract the executable from it.
+
+* If you’re using a graphic interface (e.g., GNOME or
+Unity on Ubuntu): go to your downloads directory, right
+click on the downloaded file and select _Extract Here_.
+
+* If you’re using the command line (use the
+right archive file name for your architecture):
+```
+tar -xf exercism-linux-64bit.tgz
```

Comment:
I don't think `tar` takes a `-` before the options -- did you run this and copy/paste it?

Reply:
It’s… _complicated_, as `tar` [supports](http://www.gnu.org/software/tar/manual/tar.html#SEC38) old, short _and_ long options. :)

All of those will work (and do the same thing):
```
tar xf exercism-linux-64bit.tgz
tar -xf exercism-linux-64bit.tgz
tar -x -f exercism-linux-64bit.tgz
tar --extract --file exercism-linux-64bit.tgz
```

I’m not a _huge_ fan of the old (`tar xf`) options style as it’s not symmetric with any other tool, but I totally don’t mind it either, and any of the above would work for me, so just state your preference, please. :)

(I prefer the long style in scripts, as it makes them more readable, but in case of tutorials some people dutifully type the commands rather than copy-pasting them, and the longer the options the higher the chance for a typo…)

Reply:
If the `-xf` works, then I prefer that. I had just never seen it before.